frontend/test/resolution/testFunctionCalls.cpp

```diff
@@ -132,11 +132,337 @@ static void test3b() {
   helpTest3(theFunction);
 }
 
+static void test5() {
```

Comment:
I think we should move these cases into a new test file called something like `testRuntimePrimitives.cpp`.